src/electronApp/core/dataSaver.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Save data locally
+ */
+export class DataSaver {
+  static readonly lastIPsKey = "lastIPs";
+
+  /**
+   * Saves the IP-Address into the list of last used IPs
+   * @param ip The ip address to be saved
+   */
+  public static SaveLastIP(ip: string): void {
```

Comment:
I think we should probably only store the last x IPs to avid this having boundless growth.

Reply:
I've for now limited the max IPs to 5 with 742767d